### DISCUSSION TITLE
Pinóquio 2

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#329] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/pinoquio-2/

**Comment by brunoaugusto** [comment]
[![Eu tô passada, chocada](https://i.vimeocdn.com/filter/overlay?src0=https%3A%2F%2Fi.vimeocdn.com%2Fvideo%2F1442248269-917909ad89e21509b12721c0dde266229302cb835159dd6fdbbdb76c1b268fb4-d_1280x704&src1=https%3A%2F%2Ff.vimeocdn.com%2Fimages_v6%2Fshare%2Fplay_icon_overlay.png)](https://player.vimeo.com/video/715624450)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/pinoquio-2/#comment_1142309072>):

> [![null](<https://camo.githubusercontent.com/23a1c0ff37c23a4afcbb184840b1fbf9415718e4bb5380a2164fb5b93988022d/68747470733a2f2f692e76696d656f63646e2e636f6d2f66696c7465722f6f7665726c61793f737263303d6874747073253341253246253246692e76696d656f63646e2e636f6d253246766964656f253246313434323234383236392d393137393039616438396532313530396231323732316330646465323636323239333032636238333531353964643666646262646237366331623236386662342d645f313238307837303426737263313d6874747073253341253246253246662e76696d656f63646e2e636f6d253246696d616765735f76362532467368617265253246706c61795f69636f6e5f6f7665726c61792e706e67>)](<https://player.vimeo.com/video/715624450>)

Quê que você fez pra colocar um vídeo como resposta? O ícone do editor nunca funcionou pra mim

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/pinoquio-2/#comment_1142402095>):

> Quê que você fez pra colocar um vídeo como resposta? O ícone do editor nunca funcionou pra mim

![null](<https://i.imgur.com/ik1rB0L.gif>)

**Comment by vish-blog** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/pinoquio-2/#comment_1142402095>):

> Quê que você fez pra colocar um vídeo como resposta? O ícone do editor nunca funcionou pra mim

O ícone de vídeos até funciona, o problema é uma limitação do Markdown. Daí ele combinou a sintaxe de links com a de imagens pra contornar o problema:



     ![Texto Alternativo](https://example.com/link-to-image)



E:



     [Título do Vídeo](https://example.com/my-link "Título do Vídeo")



Assim:



    [![Texto Alternativo](https://i.vimeocdn.com/filter/overlay?src0=https%3A%2F%2Fi.vimeocdn.com%2Fvideo%2F1442248269-917909ad89e21509b12721c0dde266229302cb835159dd6fdbbdb76c1b268fb4-d_1280x704&src1=https%3A%2F%2Ff.vimeocdn.com%2Fimages_v6%2Fshare%2Fplay_icon_overlay.png)](https://vimeo.com/715624450 "Título do Vídeo")



:sunglasses:

**Comment by brunoaugusto** [comment]
> [Vish!](</perfil/?user=vish-blog>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/pinoquio-2/#comment_1142430717>):

> O ícone de vídeos até funciona, o problema é uma limitação do Markdown. Daí ele combinou a sintaxe de links com a de imagens pra contornar o problema:E:Assim:😎

Estraga prazeres :stuck\_out\_tongue:
